AWSSDKSwiftCLI/Sources/AWSCLIUtils/Process+Utils.swift

```diff
@@ -42,6 +42,9 @@ public func _run(_ process: Process) throws {
     #if DEBUG
     if let testRunner = ProcessRunner.testRunner {
         try testRunner.run(process)
+        if let pipe = process.standardOutput as? Pipe {
+            try pipe.fileHandleForWriting.close()
+        }
```

Comment:
This test-only fix was necessary to ensure that CLI tests don't hang waiting for more data from stdout